libs/core/src/stories/Welcome.stories.mdx

```diff
@@ -0,0 +1,16 @@
+import { Meta, Story } from '@storybook/addon-docs';
+
+<Meta title="Welcome" parameters={{ previewTabs: { canvas: { hidden: true }, } }} />
+
+# Building with Sage
+Our system is our source of truth, providing everything you need to build the best experiences for all Kajabi entrepreneurs.
+
+## Foundations
+Our system starts with some core visual elements: Type, Color, and Icons. We also prescribe common practices for aspects of the system.
+
+## Components
+Components are the backbone of our design system. In order to be most usable, our components are written in both Rails and React form.
+Code examples, property options, and best practices can all be discovered here. Links to Figma design documentation can also be found in this section.
+
+## Contributing
+We encourage any member of the Kajabi team to feel empowered to participate. Understanding and contributing to the system are the keys to the system's longevity.
```

Comment:
Are we only building this for Kajabi? If so, lets make the repo private

cc: @monicawheeler

Reply:
So far! :nitpick

Reply:
We want our Core Design System repo (this repo) to be public facing, so our language in documentation should be geared that way.

```suggestion
We encourage everyone to feel empowered to participate. Understanding and contributing to the system are the keys to the system's longevity.
```